catalog/spatial/geocoding/spatial-geocoding-admin-module/src/main/java/org/codice/ddf/spatial/admin/module/service/Geocoding.java

```diff
@@ -32,7 +32,7 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(Geocoding.class);
 
-  private static final String BASE_DIR = System.getProperty("user.dir") + "/content/store/";
+  private static final String BASE_DIR = System.getProperty("ddf.home") + "/content/store/";
```

Comment:
:question: Not that this is part of this ticket, but I wonder how this works with when the content directory is moved since it uses BASE_DIR.